### DISCUSSION TITLE
Postman Collection API KirimWA.id

### DISCUSSION THREAD

**Discussion by microsauruz** [kirimwa-id/forum#96] [Pamerkan]
Untuk teman-teman yang ingin mencoba API KirimWA.id melalui jalur Postman,
silakan bisa menggunakan Postman Collection berikut :
[KirimWA.id.postman_collection.json.zip](https://github.com/kirimwa-id/forum/files/8052389/KirimWA.id.postman_collection.json.zip)

saya jadikan zip karena file json tidak bisa diupload ke github.
Untuk mencoba collection ini, silakan filenya di extract, lalu diimport ke postman teman-teman.
 
Terima kasih, 
Semoga membantu.

**Comment by rioastamal** [comment]
Thank you mas @microsauruz atas koleksi Postman-nya.
